x/io/int.go

```diff
@@ -51,8 +64,34 @@ func ReadVarint(data []byte) (n int64, bytesRead int, err error) {
 	return 0, 0, errValueTooLarge
 }
 
-// VarintBytes returns the number of byts a varint occupies.
+// VarintBytes returns the number of bytes required to encode a varint.
 func VarintBytes(v int64) int {
```

Comment:
Can we please add some tests for this and compare the results with the value returned from `PutVarint`? Something along the lines of:
```
for i := -1Bn; i < 1Bn; i++ {
  // compare the results of `VarintBytes` against `PutVarint`
}
```

Reply:
`PutVarint` is slow to test w/ a step size of 1. I'll add the tests that I ran manually where I test on the borders of the ranges to make sure those are correct.